about.md

```diff
@@ -6,24 +6,24 @@ permalink: about/
 
 # The beginning
 
-It all started a winter day the around the new year 2014/2015 in Stockholm, Sweden. 
-Two friends started to tinker and experiment with a game written by 
-Michael Fogleman called Craft. Our visions started to take shape and we 
-quickly realized that our take on the game was very different compared to 
+It all started a winter day the around the new year 2014/2015 in Stockholm, Sweden.
+Two friends started to tinker and experiment with a game written by
+Michael Fogleman called Craft. Our visions started to take shape and we
+quickly realized that our take on the game was very different compared to
```

Comment:
If it's not to much work, can you stop your editor from doing this? :)
Makes it harder to see what have changed.